assets/helpers/user/user.js

```diff
@@ -61,6 +64,11 @@ const init = (dispatch: Function) => {
         });
       }
     });
+  } else {
+    const emailFromSession = getEmailFromSession;
```

Comment:
what's this variable needed for? why not just call it `emailFromSession` in the first place and just do
```javascript
} else if (emailFromSession) {
  dispatch(setEmail(emailFromSession))
}

Reply:
👍